src/core/components/parameter-row.jsx

```diff
@@ -119,17 +119,21 @@ export default class ParameterRow extends Component {
       //// Find an initial value
 
       if (specSelectors.isSwagger2()) {
-        initialValue = paramWithMeta.get("x-example")
-          || paramWithMeta.getIn(["schema", "example"])
-          || (schema && schema.getIn(["default"]))
+        initialValue = [
```

Comment:
I think it would be better NOT to use array and array.find() for `initialValue`. Seems like it would be simpler to add a secondary validation check to each operation as they get called.

Reply:
update: this change is valid